parcllabs/parcllabs_client.py

```diff
@@ -9,19 +9,31 @@
 from parcllabs.services.search import SearchMarkets
 
 
-class ParclLabsClient:
-    def __init__(self, api_key: str, limit: int = 12):
-        """
-        Create a ParclLabsClient client.
+class Endpoint:
+    def __init__(self, client, url, limit):
```

Comment:
it'd be good have some docstrings to explain what/how each of these classes is doing